reacthyf/react-hyf-client/build/asset-manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
It seems like your actual homework is in the folder called `react-hyf`, but you also have this folder `reacthyf`, I think it can be removed right?